backend/copr_backend/background_worker_build.py

```diff
@@ -766,6 +857,9 @@ def build(self, attempt):
         if transfer_failure:
             raise BuildRetry("SSH problems when downloading live log: {}"
                              .format(transfer_failure))
+
+        self._keep_alive_for_user_ssh()
+        # TODO Should the results be downloaded for jobs with user SSH?
```

Comment:
If we have a reasonable file-list filter, yes.  Otherwise, NO is safer option at the beginning.